docs/2.0/integration/behave.adoc

```diff
@@ -0,0 +1,32 @@
+= Behave
+Allure integrates with behave as an external formatter.
+
+== Instalation
+[source, bash]
+----
+$ pip install allure-behave
+----
+
+== Usage
+You can specify the formatter directly in the command line:
+
+[source, bash]
+----
+$ behave -f allure_behave.formatter:AllureFormatter -o %allure_result_folder% ./features
+----
+
+== Features
+
+=== Severity
+Tags that are matched to severity names, like critical, trivial, etc, will be interpreted as feature or scenario severity.
```

Comment:
I also think this should go in parenthesis - "to severity names (like critical, trivial... etc.)" 
and also  "will be interpreted as a feature or scenario"